mkdocs.yml

```diff
@@ -0,0 +1,16 @@
+site_name: OpenShift Provider Certification Guide
+docs_dir: ./docs
+
+theme:
+  name: readthedocs
+  highlightjs: true
+  hljs_languages:
+    - yaml
+nav:
+  - 'readme.md'
```

Comment:
Any ideas on what should go in the landing page? Maybe just a few sentences copied from the main README?

Reply:
ACK, I believe so. I am adding the readme.md on this PR: https://github.com/redhat-openshift-ecosystem/provider-certification-tool/pull/23/files#diff-311fffb2f63c2d97ead8bee2b1b6ea35591871552bc00d572c3670212b043a34

I will target to the file TODO.md and it will be updated when PR #23  will be merged. Is it worked for you?

Reply:
@bostrt nvm, I just added a simple `readme.md`